### ISSUE TITLE
[ionic v4 with angular] ion-slides not emitting events

### ISSUE THREAD

**Issue by rchipman** [ionic-team/ionic#14946]
# Bug Report

**Ionic Info**

```
Ionic:

   ionic (Ionic CLI)          : 4.0.1 (C:\Users\rchipman\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework            : @ionic/angular 4.0.0-beta.0
   @angular-devkit/core       : 0.7.0-rc.3
   @angular-devkit/schematics : 0.7.0-rc.3
   @angular/cli               : 6.0.8
   @ionic/ng-toolkit          : 1.0.0
   @ionic/schematics-angular  : 1.0.1

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : browser 5.0.3

System:

   Android SDK Tools : 26.1.1
   NodeJS            : v8.11.3 (C:\Program Files\nodejs\node.exe)
   npm               : 6.2.0
   OS                : Windows 10

Environment:

   ANDROID_HOME : C:\Android\android-sdk

```

**Describe the Bug**
the `ion-slides` component is not emitting its events to the angular `EventEmitter` `@Output`s.

**Steps to Reproduce**
1. Load app
2. Slide the slide control
3. Observe the count subject display below and the console to see that the events never reached the angular controller code.

**Related Code**
Clone and run [Slides](https://github.com/rchipman/slides) project.

**Expected Behavior**
When sliding the slide control the events should be emitting.

**Comment by sbannigan** [comment]
This should be fixed by #14865

**Comment by rchipman** [comment]
This is fixed in 4.0.5 !
